src/cpu_info.rs

```diff
@@ -72,10 +72,7 @@ pub fn get_cpu_info() -> (CpuManufacturer, InstructionSet) {
         }
 
         // This ensures we always use hardware intrinsics and we do not use software emulation
-        #[cfg(all(
-            any(target_arch = "x86", target_arch = "x86_64"),
-            target_feature = "sse2"
```

Comment:
Are we sure target_feature should be removed?
If so, 'all(any(x,y), )' => 'any(x,y)'